frontend/zc_messaging/src/pages/message-board/index.jsx

```diff
@@ -112,12 +121,27 @@ const MessagingBoard = () => {
       },
       messageData: { ...newMessage }
     })
+    const newMessag = [
```

Comment:
Spell it as new message not newMessag. This misspelling may affect a future developer

Reply:
I have changed the variable to newMessages, there a variable there called newMessage